src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -1,4 +1,11 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+  <!--Uncomment the following property group when you compile .\tests\devapps\MauiApps\MauiApps.sln-->
```

Comment:
might be better to put this logic behind an environment variable like what we have on line 17 below. This way, developers can set the env var once instead of commenting and uncommenting which may cause mistakes to be merged into main.

Reply:
Actually this is implicitly done by msbuild, which will take all env variables and create properties.